packages/contracts-bedrock/snapshots/storageLayout/PermissionedDisputeGame.json

```diff
@@ -55,18 +62,11 @@
     "slot": "5",
     "type": "mapping(uint256 => bool)"
   },
-  {
-    "bytes": "1",
-    "label": "initialized",
-    "offset": 0,
-    "slot": "6",
-    "type": "bool"
-  },
   {
     "bytes": "64",
     "label": "startingOutputRoot",
     "offset": 0,
-    "slot": "7",
+    "slot": "6",
```

Comment:
Will this slot change cause slot collisions when upgrading? My guess is no since games are deployed from the factory and their implementations will not be upgraded, but want to confirm (same question about FaultDisputeGame)

Reply:
Yup. We reset the implementation when upgrading, rather than upgrade an existing implementation.

Reply:
No impact; Since the old proxies will reference the old impl, slot changes don't matter :)